src/management-system-v2/components/process-export.tsx

```diff
@@ -96,24 +118,35 @@ const ProcessExportModal: React.FC<ProcessExportModalProps> = ({ processes = [],
 
   const optionSelection = (
     <Checkbox.Group
-      options={exportSubOptions[selectedType!]}
       onChange={handleOptionSelectionChange}
       value={selectedOptions}
-      style={{ flexDirection: 'column' }}
-    />
+      style={{ width: '50%' }}
+    >
+      <Space direction="vertical">
+        {(selectedType ? exportSubOptions[selectedType] : []).map(({ label, value, tooltip }) => (
+          <Tooltip placement="left" title={tooltip} key={label}>
+            <Checkbox value={value}>{label}</Checkbox>
+          </Tooltip>
+        ))}
+      </Space>
+    </Checkbox.Group>
   );
 
   return (
     <>
       <Modal
-        title={modalTitle}
+        title={`Export selected processes`}
```

Comment:
capitalization of _processes_ (=> _Processes_)

Reply:
Fixed